### PR TITLE
Correct documentation around responders and formatters

### PR DESCRIPTION
**Pull request by shadowhand**
Documentation got a bit out of sync with recent changes and referenced
things that were no longer correct.

### CODE REVIEW

**Comment by shadowhand** [comment]
@elazar found a bunch of other docs that were out of date.

File: docs/index.md [hunk 1]
```diff
@@ -453,69 +453,17 @@ return $payload->withMessages([
...
-
-$configuration = new DefaultConfigurationSet([
-    PlatesResponderConfiguration::class,
-]);
-$configuration->apply($injector);
-```
-
-Note that this will completely replace the default group of formatters of [`FormattedResponder`](https://github.com/sparkphp/spark/blob/master/src/Responder/FormattedResponder.php). If you instead want to add to that group of formatters, you will need to use a custom configuration that modifies the response of `getFormatters()` and then invoke `withFormatters()` method to store the modified array.
-
-### Changing Responders
```
Comment (shadowhand) [inline docs/index.md]: This no longer applies because the default is set by `Action`.

File: docs/index.md [hunk 2]
```diff
@@ -453,69 +453,17 @@ return $payload->withMessages([
...
-    }
-
-    public function prepareRouter(Router $router)
-    {
-        $router->setDefaultResponder(FormatterResponder::class);
-    }
-}
-```
-
-### Route-Specific Responders
```
Comment (shadowhand) [inline docs/index.md]: Same here.

**Comment by elazar** [comment]
:+1: :star2:

**Pull request merged**